app/src/main/java/com/fstyle/androidtrainning/screen/tabhome/TabHomePresenter.java

```diff
@@ -1,15 +1,30 @@
 package com.fstyle.androidtrainning.screen.tabhome;
 
+import android.util.Log;
+import com.fstyle.androidtrainning.data.model.Movie;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.model.Movie. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports